Tests/Unit/Image/Modifier/AbstractModifierTest.php

```
@@ -0,0 +1,75 @@
+                return [];
+            }
+        };
+
+        self::assertInstanceOf(AbstractModifier::class, $modifier);
+    }
+
+    public function testInstantiationThrowsExceptionForMissingRequiredKeys(): void
+    {
+        $this->expectException(InvalidArgumentException::class);
+        
+        $modifier = new class([]) extends AbstractModifier {
```

PHPMD (2.15.0)

Avoid unused local variables such as '$modifier'. (Unused Code Rules)

(UnusedLocalVariable)